src/widget/wsearchlineedit.cpp

```diff
@@ -172,6 +173,26 @@ void WSearchLineEdit::showPlaceholder() {
     //textChanged().
     blockSignals(true);
     setText(tr("Search..."));
+    setToolTip(tr("Search" ,"noun")
+                  + "\n"
+                  + tr("Searches for a match in the current selected list (e.g. a playlist, a crate or even the whole library.\n"
+                       "Search terms can include an artist’s name, a track title, BPM, etc.\n"
+                       "Supports search operators that allow you to form more complex search queries.")
+                  + "\n\n"
+                  + tr("Shortcut")
+                  + ": \n"
```

Comment:
how does this look if you keep most terms on the same line, and only wrap when there's an actual newline in the string?  This is very long.